src/test/e2e/api/proxy/proxy.e2e.test.ts

```diff
@@ -934,7 +934,10 @@ test('Should not recursively set off timers on events', async () => {
 });
 
 test('should return all features when specified', async () => {
```

Comment:
Just for discussion: So this test does double duty as something that also checks that "strategies are correctly evaluated" when using proxy all (because the `disabledToggle` is included with `enabled: false`. Would it be worth adding that in a separate test or put that in the description somehow? Just so it's clear what it's doing? Or would it be redundant? 

In a way, it's already covered by this test, but if you're only reading test names, it's not clear that it's being tested for.

Reply:
True, I will separate it out to its own test